src/test/lrsql/admin/route_test.clj

```diff
@@ -190,6 +195,26 @@
                            :body    (u/write-json-str {"api-key"    pk
                                                        "secret-key" sk})})
                          401)))))
+    (testing "get status information"
+      (let [{:keys [status
+                    body]} (get-status headers)
+            edn-body       (u/parse-json body)]
+        ;; success
+        (is (= 200 status))
+        ;; Has data
+        (is (= {"statement-count"       0
+                "actor-count"           0
+                "last-statement-stored" nil
+                "platform-frequency"    {}
+                "timeline"              []}
+               edn-body)))
+      (testing "validates params"
+        (let [{:keys [status]} (curl/get
+                                "http://localhost:8080/admin/status?timeline-unit=foo"
+                                {:headers headers
+                                 :throw   false})]
+          ;; failure
+          (is (= 400 status)))))
```

Comment:
It would be helpful to have a test case with a valid param, just so we can compare it with the invalid case.

Reply:
_Technically_ it has one, since the lack of params is a valid params map `{}` but sure.

Reply:
https://github.com/yetanalytics/lrsql/pull/282/commits/c09ecbe58825f4f65b0b64c11631b27d50163fa3